Adds User Input Node

joshuamorton opened the pull request:
This node is simple, but important, it takes in user input and publishes it for use elsewhere.

ruyimarone committed:
basic message passing

irapha commented on buzzmobile/sense/inputer/inputer.py:
```diff
@@ -0,0 +1,16 @@
+from std_msgs.msg import String
+
+def input_node():
+    pub = rospy.Publisher('dst', String, queue_size=0)
```
can you rename 'dst' to 'destination' or something like that?

irapha commented on buzzmobile/sense/inputer/inputer.py:
```diff
@@ -0,0 +1,16 @@
+def input_node():
+    pub = rospy.Publisher('dst', String, queue_size=0)
+    rospy.init_node('inputer', anonymous=True)
+    pub.publish("")
```
IIRC, this is necessary, but can we have a comment saying why?

joshuamorton committed:
Update based on code review

joshuamorton committed:
Convert user input node to a generic repl

This way we can have a single node that handles various forms of user input for any other parts of the system.

joshuamorton commented:
Fixed
